### PR TITLE
[index] misc wording and styling improvements

### PR DESCRIPTION
**Pull request by jankeromnes**
I didn't touch the styling too much, but fixed typos and improved wordings where it made sense to me.

<details>
<summary>(old screenshot)</summary>
Screenshot_2019-03-19 Gitpod
</details>

### CODE REVIEW

File: src/pages/index.tsx [hunk 1]
```diff
@@ -55,46 +55,46 @@ export default class IndexPage extends React.Component<{}, IndexPageState> {
...
             case 0: {
                 contents = <div style={{ margin: 40 }}>
-                    <div style={{ marginTop: 60 }}>
-                        <p><strong>Just prefix any GitHub URL with <strong className="glow">gitpod.io/#</strong>.</strong></p>
-                        <p>Gitpod analysis the GitHub URL and beams you into a ready-to-code dev environment for that context.</p>
-                    </div>
                     <div style={{ margin: '40px 0px' }}>
                         <UrlAnimation />
                     </div>
+                    <div style={{ marginTop: 30 }}>
```
Comment (svenefftinge) [inline src/pages/index.tsx]: For the first item switching text and image doesn't improve IMHO.

File: src/pages/index.tsx [hunk 2]
```diff
@@ -55,46 +55,46 @@ export default class IndexPage extends React.Component<{}, IndexPageState> {
...
+                        <p>Gitpod understands the GitHub URL and beams you right into a ready-to-code dev environment for that context.</p>
+                    </div>
                 </div>;
                 break;
             }
             case 1: {
                 contents = <div style={{ margin: 40 }}>
-                    <strong>Have a button on every GitHub page.</strong>
+                    <img src={clean(BrowserExtension)} />
+                    <strong>Have a button on every GitHub page</strong>
```
Comment (svenefftinge) [inline src/pages/index.tsx]: +1 Like this better than before

File: src/pages/index.tsx [hunk 3]
```diff
@@ -55,46 +55,46 @@ export default class IndexPage extends React.Component<{}, IndexPageState> {
...
                         <button className='primary' style={{ marginLeft: 20, minWidth: 200 }}>Learn More</button>
                     </div>
-                    <img style={{ marginTop: 10 }} src={clean(BrowserExtension)} />
                 </div>
                 break;
             }
             case 2: {
                 contents = <div style={{ margin: 40 }}>
-                    <p><strong>The GitHub app builds your projects</strong> so you don't have to when starting to code.</p>
+                    <img style={{ borderRadius: 2, margin: 0, width: 700 }} src={GitHubApp} />
```
Comment (svenefftinge) [inline src/pages/index.tsx]: Here the image is bad anway. We need to see how we can improve the message and better support it visually.

File: src/pages/index.tsx [hunk 4]
```diff
@@ -55,46 +55,46 @@ export default class IndexPage extends React.Component<{}, IndexPageState> {
...
                         <button className='primary' style={{ marginLeft: 20, minWidth: 200 }}>Learn More</button>
                     </div>
-                    <img style={{ borderRadius: 2, margin: '20px 70px', width: 700 }} src={GitHubApp} />
                 </div>
                 break;
             }
         }
         return <div style={{ height: 600 }}>
-            <h2 >How It Works:</h2>
+            <h2 >Get Disposable Workspaces</h2>
```
Comment (svenefftinge) [inline src/pages/index.tsx]: We haven't talked about 'Disposable Workspaces' so far. I think we should find something simpler.

**Comment by jankeromnes** [comment]
- Rebased (new git-graph)
- Replaced "Get Disposable Workspaces" with "Get Started Coding"
- Trying with visuals between super-text and sub-text

Screenshot_2019-03-19 Gitpod(2)

**Pull request merged**